services/tasks-svc/internal/patient/events/v1/events.go

```diff
@@ -87,5 +88,11 @@ func NewPatientDischargedEvent(ctx context.Context, a hwes.Aggregate) (hwes.Even
 }
 
 func NewPatientReadmittedEvent(ctx context.Context, a hwes.Aggregate) (hwes.Event, error) {
-	return hwes.NewEvent(a, PatientReadmitted, hwes.WithContext(ctx))
+	payload := PatientReadmittedEvent{}
+	return hwes.NewEvent(a, PatientReadmitted, hwes.WithContext(ctx), hwes.WithData(payload))
+}
+
+func NewPatientDeletedEvent(ctx context.Context, a hwes.Aggregate) (hwes.Event, error) {
+	payload := PatientDeletedEvent{}
+	return hwes.NewEvent(a, PatientDeleted, hwes.WithContext(ctx), hwes.WithData(payload))
```

Comment:
This is because eventstoredb does not allow querying events which don't contain data over the eventstoredb-ui (this does not count for our custom projections). I faced this issue when extracting data for my bachelor thesis. I couldn't really find out the exact reason for that, if there is one, but I left it like this for now.

Reply:
ok, then instead we should default to an empty json object as payload in NewEvent
please make sure to add this explanation there

Reply:
oh, and is this issue know to eventstore? if not i'd be great to open an issue there

Reply:
Not yet, will do